### PR TITLE
Add a note on Nix caches

### PR DESCRIPTION
**Pull request by klntsky**

### CODE REVIEW

**Review by rynoV — APPROVED** [review]

**Review by uhbif19 — APPROVED** [review]

**Review by Bradley-Heather — APPROVED** [review]
Good idea!

File: doc/getting-started.md [hunk 1]
```diff
@@ -25,6 +26,12 @@ This guide will help you get started writing contracts with CTL. Please also see
 
 ## Prerequisites
 
+### Setting up Nix caches
+
+Nix caches improve build speeds by allowing to download pre-built dependencies instead of compiling them from source.
+
+Follow [these instructions](https://github.com/input-output-hk/cardano-ledger#nix-cache) to add the caches.
```
Comment (uhbif19) [inline doc/getting-started.md:33]: As far as I understand, without MLabs private caches there will be still many builds happening.

Comment (klntsky) [reply doc/getting-started.md:33]: I don't think so, `Push+pull caches` are set to `public-plutonomicon` in CI config.

Comment (klntsky) [reply doc/getting-started.md:33]: Obviously, it does not guarantee that we don't need to use mlabs cachix

Comment (klntsky) [reply doc/getting-started.md]: I added public-plutonomicon cache

Comment (klntsky) [reply doc/getting-started.md]: I asked what to do with mlabs cachix internally

**Pull request merged**